src/main/java/com/bina/varsim/fastqLiftover/FastqLiftOver.java

```diff
@@ -88,7 +88,7 @@ public void run(String[] args) throws IOException {
         MapBlocks mapBlocks = new MapBlocks(mapFile);
 
         if (fastqType.equals("dwgsim")) {
-            final File outFile = (outFiles != null && outFiles.size() == 1) ? outFiles.get(0) : null;
+            final File outFile = (outFiles != null && outFiles.size() == 1) ? outFiles.get(0) : null; // marghoob this is not used?
```

Comment:
Remove it. It was used at some point but I forgot to remove it after code restructuring.